FWCore/Skeletons/python/utils.py

```diff
@@ -20,6 +20,20 @@
 # template tag pattern
 TAG = re.compile(r'[a-zA-Z0-9]')
 
+def template_directory():
+    "Return location of template directory"
+    mkTemplates = "src/FWCore/Skeletons/mkTemplates"
+    # Check developer area first
+    if "CMSSW_BASE" in os.environ:
+        ret = os.path.join(os.environ["CMSSW_BASE"], mkTemplates)
+        if os.path.exists(ret):
+            return ret
+    # Then release area
+    ret = os.path.join(os.environ["CMSSW_RELEASE_BASE"], mkTemplates)
```

Comment:
Does this work with patch releases?

Reply:
Yes, https://github.com/cms-sw/cmssw/issues/29306#issuecomment-624120098
> Patch release contians symlinks for missing packages.